JASP-Engine/JASP/R/equivalencebayesianindependentsamplesttest.R

```diff
@@ -152,7 +152,9 @@ EquivalenceBayesianIndependentSamplesTTest <- function(jaspResults, dataset, opt
   if (ready)
     equivalenceBayesianIndTTestTable$setExpectedSize(length(options$variables))
 
-  message <- gettextf("I ranges from %1$s to %2$s", options$lowerbound, options$upperbound)
+  message <- gettextf("I ranges from %1$s to %2$s", 
+                      ifelse(options$lowerbound == -Inf, "-\u221E", options$lowerbound),
```

Comment:
```suggestion
                      if(options$lowerbound == -Inf) "-\u221E" else options$lowerbound),
```
same goes for almost all other calls to `ifelse`. Although `ifelse` is slightly shorter, it's doing a lot more and is not really an appropriate ternary operator.

Reply:
Also, perhaps this particular `gettextf` statement should be a function inside `commonequivalencefunctions.R` since the statement is reused three times. Perhaps something like:

```r
.equivalenceGetIntervalMessage(lowerbound, upperbound) {
  message <- gettextf("I ranges from %1$s to %2$s", 
                      if(lowerbound == -Inf), "-\u221E", lowerbound),
                      if(upperbound == -Inf), "\u221E",  upperbound))
  message
}
```
Duplicating the code also introduces three identical translation entries for `"I ranges from %1$s to %2$s"`.